api/src/tests/transactions_test.rs

```diff
@@ -337,8 +337,8 @@ async fn test_multi_ed25519_signed_transaction() {
     context.commit_block(&vec![create_account_txn]).await;
 
     let raw_txn = factory
-        .mint(auth_key.derived_address(), 1000)
-        .sender(auth_key.derived_address())
+        .mint(auth_key.account_address(), 1000)
```

Comment:
What's the difference?

Reply:
I found this name confusing, hard to find, and implies something that isn't accurate.